packages/ocaml-systemd/ocaml-systemd.1.3/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "Juergen Hoetzel <juergen@archlinux.org>"
+authors: "Juergen Hoetzel <juergen@archlinux.org>"
+homepage: "https://github.com/juergenhoetzel/ocaml-systemd/"
+bug-reports:  "https://github.com/juergenhoetzel/ocaml-systemd/issues"
+dev-repo: "git+https://github.com/juergenhoetzel/ocaml-systemd.git"
+license: "LGPL-3.0-only WITH OCaml-LGPL-linking-exception"
+
+build: [
+  "dune"
+  "build"
+  "-j"
+  jobs
+  "@install"
+]
+depends: [
+  "ocaml"
+  "dune" {build & >= "2.0"}
+]
+depexts: [
+  ["libsystemd-dev"] {os-family = "debian"}
+  ["systemd-devel"] { os-family = "suse" | os-family = "opensuse" | os-family = "fedora"}
+]
```

Comment:
In the future it would be nice to replace this by a `conf-libsystemd` package to make it easier to maintain.